rfcs/0078-configuration-file-generators.md

```diff
@@ -0,0 +1,142 @@
+--- feature: configuration_file_generators start-date: 2020-10-18 author:
```

Comment:
Looks like auto-formatting broke this header?

Reply:
Thanks, fixed